pkg/sql/schemachanger/scpb/elements.proto

```diff
@@ -484,6 +488,13 @@ message IndexColumn {
 
   // Direction is only populated for KEY columns.
   sql.catalog.catpb.IndexColumn.Direction direction = 6;
+
+  // Indicates if this column is implicit.
```

Comment:
nit: worth noting that this relates to partitioning.

Reply:
Good point:

  // Indicates if this column is implicitly included within the index for
  // partitioning.